Competition2017/src/competition/subsystems/shooter_belt/commands/RunBeltIfWheelAtSpeedCommand.java

```diff
@@ -0,0 +1,37 @@
+package competition.subsystems.shooter_belt.commands;
+
+import org.apache.log4j.Logger;
+
+import competition.subsystems.shooter_belt.ShooterBeltSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class RunBeltIfWheelAtSpeedCommand extends BaseCommand {
+
+    private static Logger log = Logger.getLogger(RunBeltIfWheelAtSpeedCommand.class);
+    
+    final ShooterBeltSubsystem beltSubsystem;
+    public ShooterWheelSubsystem shooterWheelSubsystem;
+    final double targetBeltSpeed;
+    
+    public RunBeltIfWheelAtSpeedCommand(ShooterBeltSubsystem beltSubsystem){
+        this.beltSubsystem = beltSubsystem;
+        this.requires(beltSubsystem);
+        targetBeltSpeed = beltSubsystem.getBeltTargetSpeed();
```

Comment:
Agree this is awkward for the belt, there's a task to refactor it to be less insane but I think when we do John's new base subsystem it will take care of it anyway: https://app.asana.com/0/38541457243752/270337401932322